styles/style.css

```diff
@@ -6,3 +6,189 @@ Green: #00B172
 White: #FFF
 
 */
+
+html, body {
+    font: "Sans Serif";
+    margin: 0;
+    padding: 0;
+  }
+  
+
+  h1 {
+    font-size: 9em;
+    color: #FFF;
+    text-align: center;
+  }
+
+  body {
+    font-family: "Sans Serif";
+    font-size: 10px;
+    line-height: 3.5em;
+    font-weight: 300;
+  }
+
+/* |||||||| Navbar |||||||| */
```

Comment:
Is this comment necessary? The same question goes for the other comments.

Reply:
Not necessary, no.

I am using this type of  comment to differentiate the  sections. It's a visual help.